data_browser/src/utils.js

```diff
@@ -1,19 +1,30 @@
+function getPrefix() {
```

Comment:
@epicfaace There's definitely a better way to do this, but just hacking this for now...

Reply:
Links from the marin.community have the marin-data-browser.*run.app prefix because we're inside the iframe